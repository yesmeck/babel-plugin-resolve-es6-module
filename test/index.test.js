import { transformFileSync } from 'babel-core';
import path from 'path';
import plugin from '../src';

describe('babel-plugin-pkg-module', () => {
  it('repalce cmd module with es6 module', () => {
    const result = transformFileSync(path.resolve(__dirname, './fixture/src/index.js'), {
      plugins: [[plugin, {
        'rc-util': {
          mainDir: 'lib',
          moduleDir: 'es',
        },
      }]],
      babelrc: false,
    });
    expect(result.code).toMatchSnapshot();
  });
});
