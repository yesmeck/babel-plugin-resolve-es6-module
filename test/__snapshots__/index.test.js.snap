// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-plugin-pkg-module repalce cmd module with es6 module 1`] = `
"import deprecated from 'rc-util/es/deprecated';
import emptyFunction from 'fbjs/lib/emptyFunction';
import ScrollableInkTabBar from 'rc-tabs/es/ScrollableInkTabBar';
import Table from 'rc-table';
import foo from '../foo';
import bar from './bar';

export { guid } from 'rc-util/es/guid';
export { foo };"
`;
